pages/docs/subscription.en-US.mdx

```diff
@@ -0,0 +1,82 @@
+import { Callout } from 'nextra-theme-docs'
+
+# Subscription
+
+<Callout emoji="✅">
+  Please update to the latest version (≥ 2.1.0) to use this API.
+</Callout>
+
+## `useSWRSubscription`
+
+`useSWRSubscription` is a React hook that allows subscribing to real-time data sources with SWR. Here is the API documentation for the code you provided:
+
+```tsx
+useSWRSubscription(key: any, subscribe: Function): { data: any, error: Error | null }
+```
+
+### API
+
+This hook subscribes to a real-time data source using the subscribe function provided, and returns the latest data received and any errors encountered. The hook automatically updates the returned data as new events are received.
+
+#### Parameters
```

Comment:
I think it has to be clear that these are arguments passed to the subscribe function, not the hook